src/tests/ftest/launch.py

```diff
@@ -809,6 +809,10 @@ def run_tests(test_files, tag_filter, args):
     avocado_logs_dir = os.path.expanduser(avocado_logs_dir[0])
     print("Avocado logs stored in {}".format(avocado_logs_dir))
 
+    # Remove stale job results, if any, before test run
+    if os.path.isdir(avocado_logs_dir):
+        rmtree(avocado_logs_dir)
+
```

Comment:
We shouldn't be doing this here.  When developers are running launch.py manually I don't think they'll appreciate all of their previous test results being removed.

In CI this should be mandatory, but it either belongs further up in the script hierarchy (e.g. `ftest.sh` or in `ci/functional/test_main.sh`) or as a conditional operation driven by a `launch.py` command line option.

Reply:
ok, good suggestion. thanks.